konfera/event/views.py

```diff
@@ -127,3 +128,13 @@ def event_public_tickets(request, slug):
 
     context['tickets'] = available_tickets
     return render(request=request, template_name='konfera/event_public_tickets.html', context=context)
+
+
+def event_order_detail(request, slug, order_uuid):
+    context = dict()
+    event = get_object_or_404(Event.objects.published(), slug=slug)
```

Comment:
@iraquitan great work the order looks very nice, and the label is great idea.

I think to in reality we will always have only tickets from the same event in one order, besides it occurs to me as a better solution. But somewhere on the receipt should the event be shown, doesnt have to be in ordered tickets section, it can be in order details, or somewhere where do you think is most relevant. 

@ZoltanOnody thanks for the review, and the logic how it will be used, I have explained [here](https://github.com/pyconsk/django-konfera/issues/173#issuecomment-255449895). I think the property decorator is the right thing to do.